sql/alps_submitter.go

```diff
@@ -329,30 +381,185 @@ func generateEstimatorCreator(estimator string, attrs []*attribute) (string, err
 	return fmt.Sprintf("tf.estimator.%s(%s)", estimator, strings.Join(cl, ",")), nil
 }
 
-func genALPS(w io.Writer, pr *extendedSelect, fts fieldTypes, db *DB) error {
-	// TODO(uuleon): codegen alps
+func newAlpsFiller(pr *extendedSelect) (*alpsFiller, error) {
+	fcList, fsMap, err := resolveTrainColumns(&pr.columns)
```

Comment:
Will Alps support prediction/inference soon? 

If so, we might want to add a check here

```go
if pr.training {
```

If not, we might not need the function `submitALPS` in addition to `trainALPS`.

Reply:
Yes, ALPS will support inference in the next few days. This check already exists in the `submitALPS` function. To avoid misunderstanding, what do you think about renaming `newAlpsFiller` to `newALPSTrainerFiller`?